be/src/storage/delta_writer.cpp

```diff
@@ -244,11 +230,8 @@ Status DeltaWriter::close() {
                                                  _state_name(state)));
     case kClosed:
```

Comment:
1. It seems the kClosed state is redundant.
2. And add a initialized -> committed state transition can avoid the rollback log.

Reply:
1. kClosed is used to flush last memtable, it is useful.
2. how?